
  -> R code for visualization is included in "RScript"

  -> The "rds_ge_ml" folder gives the results of statistical analyses
  
  -> Clinical metadata: metadata.csv (variables: "ID" relate to genome sequence; 
  "collection_data_gsheet, vax_1_date, vax_2_date, booster_1_date, booster_2_date" calculate vaccination status;
  "lineage" define the variants of concern;
  "yale_n1_ge_ml" is viral copies).
   
  -> GISAID data: https://gisaid.org/

  -> All genome sequences used for the GWAS analysis and associated metadata in this dataset are published in GISAIDâ€™s EpiCoV database. To view the contributors of each individual sequence with details such as accession number, Virus name, Collection date, Originating Lab and Submitting Lab and the list of Authors, visit https://epicov.org/epi3/epi_set/240219fh?main=true
  
 
